automation/_input/04_Powers_SAMPLE.yaml

```diff
@@ -95,6 +95,114 @@ Momentum: # Without this power, this should cost a major action.
         Role: Defender
         Skill: Brute > 1
 
+Attack, Mystic:
```

Comment:
What motivated including these in the sample?

Reply:
Currently only Defender and Martial are supported in the SAMPLE, adding additional powers gives a fuller representation of Powers a Support and Caster could use.

Reply:
I propose we reserve the sample for items that test the bounds of what the code can handle, edge cases. That might include items that shouldn't actually be in the game (e.g., what if a power had a save and a stat override)